contracts/tenderizer/Tenderizer.sol

```diff
@@ -135,6 +159,14 @@ abstract contract Tenderizer is Initializable, ITenderizer, SelfPermit {
         _withdraw(address(this), _unstakeLockID);
     }
 
+    function migrateWithdraw(uint256 _unstakeLockID) external onlyGov {
+        // reset pending migration amount
+        assembly {
+            sstore(PENDING_MIGRATION_SLOT, 0)
+        }
+        _withdraw(address(this), _unstakeLockID);
```

Comment:
We should consider rebasing after the withdraw to reduce call order dependency. 

This would be helpful for Livepeer as well as it would allow the migration for Livepeer to happen atomically